packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -399,6 +385,36 @@ const DMT5CompareModalContent = ({
         return isDesktop() ? 'xxxs' : 'xxxxs';
     };
 
+    const is_account_added = (item: TFooterButtonData) =>
```

Comment:
Function names should be in camelCase as per our JS guideline.
https://github.com/binary-com/deriv-app/blob/master/docs/Stylesheet/README.md